TestProjects/iOS-Support/XamarinSidebar/MvvmCross.iOS.Support.Sidebar/Views/CenterPanelView.cs

```diff
@@ -42,7 +42,7 @@ public override void ViewDidLoad()
             var bindingSet = this.CreateBindingSet<CenterPanelView, CenterPanelViewModel>();
             bindingSet.Bind(label).To(vm => vm.ExampleValue);
             bindingSet.Bind(rightPanelInstructions).To(vm => vm.RightPanelInstructions);
-            bindingSet.Bind(masterButton).To(vm => vm.ShowMasterCommand);
+			bindingSet.Bind(masterButton).For("TouchDown").To(vm => vm.ShowMasterCommand);
```

Comment:
I believe this PR would not affect the default binding for a [UIButton](https://github.com/MvvmCross/MvvmCross/blob/develop/MvvmCross/Binding/iOS/MvxIosBindingBuilder.cs#L191). It would just default to the `TouchUpInside` version of this custom binding.